blueocean-web/src/main/resources/io/jenkins/blueocean/TryBlueOceanPageDecorator/header.jelly

```diff
@@ -0,0 +1,8 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler">
+  <!--
+  The following Stapler adjunct adds the "Try Blue Ocean JavaScript bundle to
+  every page in "classic" Jenkins.
+  -->
+  <st:adjunct includes="io.jenkins.blueocean.try"/>
```

Comment:
Sorry ... brain fart there ... we'll still need the JavaScript anyway. The `PageDecorator` is just a hook, it's the JavaScript that lets you get the element (`<a>` or `<div>` ... mostly irrelevant) into the code, so you're going to need that anyway => amount of code ~ same.